product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -4,25 +4,44 @@ redirects:
   - 02_high_availibility
 ---
 
-BigAnimal enables deploying a cluster with or without high availability. The option is controlled with the **High Availablity** control on the [Create Cluster](https://portal.biganimal.com/create-cluster) page in the [BigAnimal](https://portal.biganimal.com) portal.    
+BigAnimal supports three deployment architectures: 
+- Single node
+- Standard high availability
+- Extreme high availability (beta)
 
-## High availability—enabled
+You choose which type of deployment you want when creating your cluster on the [Create Cluster](https://portal.biganimal.com/create-cluster) page in the [BigAnimal](https://portal.biganimal.com) portal.  
+
+## Single node
+
+For nonproduction use cases where high availability is not a primary concern, a cluster deployment with high availability not enabled provides one primary with no standby servers for failover or read-only workloads. 
+
+In case of permanent unavailability of the primary, a restore from a backup is required. 
+
+![*BigAnimal Cluster4*](images/Single-Node-Diagram-2x.png)
+
+## Standard high availability
 
 The high availability option is provided to minimize downtime in cases of failures. High-availability clusters—one *primary* and two *replicas*—are configured automatically, with replicas staying up to date through physical streaming replication. In cloud regions with availability zones, clusters are provisioned across multiple availability zones to provide fault tolerance in the face of a datacenter failure.
```

Comment:
Maybe we should make it clear that the availability will be in the same region. For example:

"If supported by the cloud region, clusters are provisioned across multiple availability zones in that region."

Reply:
That doesn't read clearer to me - I'd just remove "multiple availability" from the second half of the sentence. The rest of the information re: fault tolerance is important.